### ISSUE TITLE
Travis Failure

### ISSUE THREAD

**Issue by blink1073** [scikit-image/scikit-image#1692]
We seem to be timing out on the 2.6 builds on this test, any thoughts?

```
skimage.measure.tests.test_regionprops.test_dtype ... /home/travis/venv/lib/python2.6/site-packages/scipy/ndimage/measurements.py:270: DeprecationWarning: integer argument expected, got float
  return _nd_image.find_objects(input, max_label)
```

**Comment by jwiggins** [comment]
So if you look at the code for that test, I'm pretty sure the timeout is happening here: https://github.com/scikit-image/scikit-image/blob/master/skimage/measure/tests/test_regionprops.py#L35-L36
When I try doing what `regionprops()` does in ipython (python 2.7.10, scipy 0.16.0, numpy 1.9.2), I get a `TypeError` for the equivalent of this line: https://github.com/scikit-image/scikit-image/blob/master/skimage/measure/_regionprops.py#L496. That's what the test is expecting too.

The fact that a `DeprecationWarning` log message is being printed here is very suspicious. That line (https://github.com/scipy/scipy/blob/v0.16.0/scipy/ndimage/measurements.py#L270) is a call into the `_ndimage` C-extension. None of the other builds get that log message.

So, uhh... Gremlins? Yeah, my money is on Gremlins. :dragon_face:

**Comment by blink1073** [comment]
Fixed by #1694.
